build/commands/lib/config.js

```diff
@@ -732,11 +732,11 @@ Config.prototype.buildArgs = function () {
 
     args.brave_ios_developer_options_code = this.braveIOSDeveloperOptionsCode
 
-    // This is currently being flipped on and off by the Chromium team to test
-    // however it causes crashes for us at launch. Check `ios/features.gni`
-    // in the future to see if this is no longer needed
-    // https://github.com/brave/brave-browser/issues/29934
-    args.ios_partition_alloc_enabled = false
+    // The app currently crashes on launch without disabling these 2 args:
+    // https://github.com/brave/brave-browser/issues/49595
```

Comment:
the linked upstream issue is in a "fixed" state already. Are you sure we have the same issue?